src/style.css

```diff
@@ -1,3 +1,92 @@
 body {
   margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+
+  --link-hover-color: #00acdc;
+  --link-active-color: var(--link-hover-color);
+  --header-max-width: 1200px;
+  --header-min-width: 1024px;
+  --header-bckgrd-color: #fff;
+  --header-line-height: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  max-width: var(--header-max-width);
+  min-width: var(--header-min-width);
+  margin: 0 auto;
+  background-color: var(--header-bckgrd-color);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: auto 0 auto 50px;
+}
+
+.logo__img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  margin-left: 20px;
+  line-height: var(--header-line-height);
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: #000;
+}
+
+/* LoVeHAte */
+.nav__link:hover {
+  color: var(--link-hover-color);
+}
+
+.nav__link:active {
+  color: var(--link-active-color);
+}
+
+.nav__link:active::after {
+  position: absolute;
+  content: "";
+  height: 4px;
+  width: 100%;
+  bottom: 0;
+  left: 0;
+  background-color: var(--link-active-color);
+  border-radius: 8px;
+}
+
+.nav__item:first-child .nav__link {
+  margin-left: 0;
+}
```

Comment:
do not increase specifity
delete margin-left from `nav__link`  and  try instead
```
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```